Simplified/NYPLReaderReadiumView.m

```diff
@@ -285,10 +286,8 @@ - (void)applyCurrentFlowIndependentSettings
                                                    ",  [NYPLReaderSettings sharedSettings].backgroundMediaOverlayHighlightColor.javascriptHexString];
     
     [self sequentiallyEvaluateJavaScript:javascriptToChangeHighlightColour];
-    
     self.backgroundColor = [NYPLReaderSettings sharedSettings].backgroundColor;
     self.webView.backgroundColor = [NYPLReaderSettings sharedSettings].backgroundColor;
-    [self sequentiallyEvaluateJavaScript:@"simplified.settingsDidChange();"];
```

Comment:
Just a couple random drive-by questions here.

1. Why is settingsDidChange being removed? Based on the name, that seems like a good place to put the code you put into updateCFI.

Reply:
I think originally `settingsDidChange` did more than just update the CFI so I renamed the function, too.